packages/vscode-extension/src/officeChat/utils.ts

```diff
@@ -30,11 +36,19 @@ export async function purifyUserMessage(
     new LanguageModelChatMessage(LanguageModelChatMessageRole.User, userMessagePrompt),
     new LanguageModelChatMessage(LanguageModelChatMessageRole.System, systemPrompt),
   ];
+  const t0 = performance.now();
   const purifiedResult = await getCopilotResponseAsString(
     "copilot-gpt-4",
     purifyUserMessage,
     token
   );
+  const t1 = performance.now();
+  const requestTokens = countMessagesTokens(purifyUserMessage);
+  const responseTokens = Tokenizer.getInstance().tokenLength(purifiedResult);
```

Comment:
Use countMessagesTokens here. Same for the rest.

Reply:
done, thanks